cinema/serializers.py

```diff
@@ -1 +1,80 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Movie, Actor, Genre, CinemaHall, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
```

Comment:
Isn't this line too long?

Reply:
It is exactly 79 characters long. Otherwise, it wouldn't have passed the flake8 checks on GitHub

Reply:
If the line suits then keep it